### DISCUSSION TITLE
jicodda engraver set up

### DISCUSSION THREAD

**Discussion by carlycraig** [arkypita/LaserGRBL#2350] [Q&A]
I bought this stuff for my wife in july and i cant get it to work! Please hlp or suggestions

**Comment by Tribo-2dot0** [comment]
Primeiro leia tudo aqui com calma e veja como instalar o driver CH340, necessário para que haja comunicação entre a máquina e o computador: https://lasergrbl.com/usage/
Depois esse vídeo pode auxiliar em algum entendimento também: www.youtube.com/watch?v=OhNsO9G06lk
